Kernel/Bus/USB/Drivers/HID/MouseDriver.cpp

```diff
@@ -66,6 +68,12 @@ ErrorOr<void> MouseDriver::initialize_device(USB::Device& device, USBInterface c
 
     auto const& endpoint_descriptor = interface.endpoints()[0];
     auto interrupt_in_pipe = TRY(USB::InterruptInPipe::create(device.controller(), device, endpoint_descriptor.endpoint_address, endpoint_descriptor.max_packet_size, 10));
+
+    // Switch to the boot protocol. By default the report protocol is used (see 7.2.6 Set_Protocol Request).
```

Comment:
Can you say "why?" in addition to "what?"

Reply:
We only support the boot protocol. I added that as a reason to the comment now.